assets/src/edit-story/components/canvas/gridview/index.js

```diff
@@ -133,6 +84,8 @@ function RangeControl( { value, onChange } ) {
 				step="1"
 				value={ value }
 				onChange={ ( evt ) => onChange( Number( evt.target.value ) ) }
+				width="360px"
```

Comment:
I'd rather have this as `width={360}` and add `px` inside the component.

Reply:
this is to allow for percentage values, if we don’t think that’s useful then sure

Reply:
What about always using 100% and let the parent element dictate the max width?